types/inline-style-prefixer/index.d.ts

```diff
@@ -4,6 +4,8 @@
 //                 dpetrezselyova <https://github.com/dpetrezselyova>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+type CSSStyleDeclaration = any;
```

Comment:
this type is already defined in the default library. make sure to include `--lib dom` in your compilation.